vm/2.0/modules/ROOT/pages/index.adoc

```diff
@@ -3,75 +3,57 @@
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-VM Connector v2.0
+Anypoint Connector for Virtual Machine (VM Connector) handles intra-app and inter-app communication through asynchronous queues that can be transient or persistent.
 
-Anypoint Connector for Virtual Machine (VM) (VM Connector) handles intra-app and inter-app communication through asynchronous queues that can be transient or persistent.
+* Transient queues are faster than persistent queues, but they are not reliable in the case of a system crash
+* Persistent queues are slower but reliable
 
-* Transient queues are faster than persistent queues, but they are not reliable in the case of a system crash.
-* Persistent queues are slower but reliable.
+When running on a single instance, persistent queues work by serializing and storing the contents on the disk. However, when running in cluster mode, the memory grid backs the persistent queues. Thereby, when a flow uses the VM Connector to publish content to a queue, Mule runtime engine determines whether to process the message in the same origin node or to send the message to the cluster to pick another node. The VM Connector facilitates the load distribution across a cluster.
```

Comment:
Maybe the info would make more sense to me ordered like this (if it is technically accurate):

VM Connector facilitates communication that uses either transient and persistent data queues:
* Transient queues process data more quickly, but are more susceptible to data loss if the system crashes.
   (Text that tells why, and what roll the connector plays with transient data queues)
* Persistent queues process data more slowly but are less susceptible to data loss.
   There are two scenarios: single-instance processing and cluster-mode operation:
          ** Single
               How it works, what role connector plays
          ** Cluster
               How it works, what role connector plays
     In either case, data in persistent queues must be serializable. Although you can use Kyro to meet this
     requirement, it is limited <in this way>. It is better <to not use Kyro, or to use Kyro with these additional
     requirements>:
         ** Simple
         ** Serializable interface....

Reply:
OR
Maybe we should have separate subtopics rather than two bullets, because the persistent data queue scenario is so much more complicated than the transient:

...transient or persistent data queues

==Communication Using a Transient Data Queue
as above

==Communication Using a Persistent Data Queue
as above

Reply:
Questions about existing text:
```suggestion
When running on a single instance, persistent queues work by serializing and storing the contents on the disk. However, when running in cluster mode, the memory grid backs the persistent queues. Thereby, when a flow uses the VM Connector to publish content to a queue, Mule runtime engine determines whether to process the message in the same origin node or to send the message to the cluster to pick another node. The VM Connector facilitates the load distribution across a cluster.
```
I don't understand the chronology. This text makes it seem...:
1. How single instance processing works
2. How cluster mode processing works
3. How Mule decides which whether to use single instance (same node) or cluster mode (distributed)???
4. VM Connector helps only with cluster mode?

Reply:
I can't really edit this above text, because the order of information is not clear to me.
Maybe another connector peer writer can help, using my questions?